src/js/components/viewer/random-visual-player.jsx

```diff
@@ -251,12 +233,12 @@ var RandomVisualPlayer = React.createClass({
             };
 
             return (
-                <MediaObject ref={mediaObject.guid} key={mediaObject.guid} data={data}></MediaObject>
+                <MediaObject ref={mediaObject.guid} key={mediaObject.guid} data={data} />
             );
         });
 
         return (
-            <div className="player" ref="player">
+            <div className="player" ref="player" style={this.props.sceneStyle}>
```

Comment:
Use the style property for react to set our style.  This implements all the logic removed from this component.